### ISSUE TITLE
Complete dataset and key parameters

### ISSUE THREAD

**Issue by MrShouxingMa** [Linxyhaha/EQUAL#2]
This is a very interesting piece of work that considers the cold start problem from the perspective of feature migration.

However, when I run the code, both datasets you provided do not run successfully, where it seems micro-video is missing key files such as validation_dict.npy; in addition, the Amazon dataset will run incorrectly due to inconsistencies in parameter inputs causing runtime errors in the calculations;

Would you mind double-checking that the key parameters of the uploaded code are consistent with the paper that
In other words, can you successfully run the code that has been uploaded?

Also, would you mind uploading the full micro-video dataset to facilitate community research?

Looking forward to your reply！😘

**Comment by Linxyhaha** [comment]
Hi! Thanks for your interest in our work! For the complete dataset, you can access them in repository of [TDRO](https://github.com/Linxyhaha/TDRO). For the code reproduction, we will check as soon as possible and update the codes promptly. Thanks for your patience :)
